src/client/replication_ddl_client.h

```diff
@@ -312,28 +354,52 @@ class replication_ddl_client
         return task;
     }
 
+    // The same as the above, except that the thread hash for the RPC response task is set to 0.
+    template <typename TRequest>
+    rpc_response_task_ptr request_meta(const dsn::task_code &code,
+                                       std::shared_ptr<TRequest> &req,
+                                       int timeout_milliseconds)
+    {
+        return request_meta(code, req, timeout_milliseconds, 0);
+    }
+
+    // The same as the above, except that the timeout for the RPC request is set to 0.
```

Comment:
Add code annotation to explain what does `0` mean to the function with `timeout_milliseconds` parameter. (never timeout?)

Reply:
OK, added the comments to describe.